app/templates/components/question-single.hbs

```diff
@@ -0,0 +1,14 @@
+<h3>{{question.title}}</h3>
+<ul class="list-unstyled">
+  {{#each question.options as |option|}}
+    {{#if (if-eq question.type "input")}}
+        <li class="question-answer">
+          {{answer-tile/input updateTag=(route-action "updateTag" member chapter question option tag) option=option tag=tag}}
+        </li>
+    {{else}}
+        <li class="question-answer">
+          {{answer-tile saveTag=(route-action "saveTag" member chapter question option tag) clearTag=(route-action "clearTag" tag option) question=question option=option tag=tag}}
```

Comment:
In the answer-tile component, only question and tag are needed. Remove other arguments sent to component.